[1.0.3] Distribution Commit Build Fail

Trying to build from commit: 9bfbb76e57fa598ced9117e53b65b06028eeb920 throws:

Error: Cannot find module './utils/uniffe.js'

as gulpfile.js:27:14 is requiring this file not included in the branch.

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -657,7 +657,8 @@ gulp.task('templates:mdl', function() {
 
 gulp.task('_release', function() {
   return gulp.src(['dist/material?(.min)@(.js|.css)?(.map)', 'LICENSE',
-    'README.md', 'bower.json', 'package.json', './sr?/**/*', 'gulpfile.js'])
+    'README.md', 'bower.json', 'package.json', '.jscsrc', '.jshintrc',
+    './sr?/**/*', 'gulpfile.js', './util?/**/*'])
     .pipe(gulp.dest('_release'));
 });
 

